vcluster_versioned_docs/version-0.20.0/configure/vcluster-yaml/control-plane/components/host-path-mapper.mdx

```diff
@@ -15,11 +15,32 @@ Virtual cluster internal logging relies on a separate component called the [Host
 
 If you don't map host paths, logs for vCluster pods are incorrectly resolved by log collectors such as Loki, ELK, and Fluentd.
 
-## vCluster open source
+## vCluster Open Source
 
-Set `enabled: true` to configure vCluster to deploy a HostPath Mapper DaemonSet per Node per vCluster instance.
+### Enabling the HostPath Mapper
+Set `enabled: true` in the vCluster config to allow it to use the HostPath Mapper daemonset, which would be created in the next step.
+```YAML title="vcluster.yaml"
+controlPlane:
+  hostPathMapper:
+    enabled: true
+```
 
-## vCluster pro
+### Deploy the HostPath Mapper daemonset
+Now that the vCluster itself is ready, the HostPath Mapper component can be deployed. The following 2 pieces of information are needed for this:
+* The HostPath Mapper has to be deployed in the same namespace as the target vCluster
+* The `.Values.VclusterReleaseName` value when deploying this Helm Chart needs to be equal to the name of the target vCluster
+
+For example if your vCluster is named `my-vcluster` and is deployed in namespace `my-namespace` then you should run
+```YAML title="vcluster.yaml"
```

Comment:
This is not yaml, but bash.
Also title does not line up with the command at hand.

Reply:
Ah. My bad.